examples/pysdl2.py

```diff
@@ -0,0 +1,357 @@
+                        "windows_key_code": keycode,
+                        "character": keycode,
+                        "unmodified_character": keycode,
+                        "modifiers": cef.EVENTFLAG_NONE
+                    }
+                    browser.SendKeyEvent(key_event)
+                elif event.key.keysym.sym in [
+                        sdl2.SDLK_BACKSPACE,
+                        sdl2.SDLK_DELETE,
+                        sdl2.SDLK_LEFT,
+                        sdl2.SDLK_RIGHT,
+                        sdl2.SDLK_UP,
+                        sdl2.SDLK_DOWN,
+                        sdl2.SDLK_HOME,
+                        sdl2.SDLK_END
+                ]:
+                    keycode = getKeyCode(event.key.keysym.sym)
+                    key_event = {
+                        "type": cef.KEYEVENT_RAWKEYDOWN,
+                        "windows_key_code": keycode,
+                        "character": keycode,
+                        "unmodified_character": keycode,
+                        "modifiers": cef.EVENTFLAG_NONE
+                    }
+                    browser.SendKeyEvent(key_event)
+            elif event.type == sdl2.SDL_KEYUP:
+                # Handle key up events for non-text keys
+                if event.key.keysym.sym in [
+                        sdl2.SDLK_RETURN,
+                        sdl2.SDLK_BACKSPACE,
+                        sdl2.SDLK_DELETE,
+                        sdl2.SDLK_LEFT,
+                        sdl2.SDLK_RIGHT,
+                        sdl2.SDLK_UP,
+                        sdl2.SDLK_DOWN,
+                        sdl2.SDLK_HOME,
+                        sdl2.SDLK_END
+                ]:
+                    keycode = getKeyCode(event.key.keysym.sym)
+                    key_event = {
+                        "type": cef.KEYEVENT_KEYUP,
+                        "windows_key_code": keycode,
+                        "character": keycode,
+                        "unmodified_character": keycode,
+                        "modifiers": cef.EVENTFLAG_NONE
+                    }
+                    browser.SendKeyEvent(key_event)
+        # Clear the renderer
+        sdl2.SDL_SetRenderDrawColor(
+            renderer,
+            backgroundColour.r,
+            backgroundColour.g,
+            backgroundColour.b,
+            255
+        )
+        sdl2.SDL_RenderClear(renderer)
+        # Tell CEF to update which will trigger the OnPaint
+        # method of the RenderHandler instance
+        cef.MessageLoopWork()
+        # Update display
+        sdl2.SDL_RenderCopy(
+            renderer,
+            renderHandler.texture,
+            None,
+            sdl2.SDL_Rect(0, headerHeight, browserWidth, browserHeight)
+        )
+        sdl2.SDL_RenderPresent(renderer)
+    # User exited
+    exit_app()
+    
+def getKeyCode(key):
+    """Helper function to convert SDL2 key codes to cef ones"""
+    if key == sdl2.SDLK_RETURN:
```

Comment:
This mapping could be done using a dict:
```
map_keys = {
    sdl2.SDLK_RETURN: 13,
    sdl2.SDLK_DELETE: 46,
    ...
]
```
Key mappings is incomplete, so this code will probably be extended by user, so let's get rid of those ifs to make it more clear.